packages/presentation/src/file.ts

```diff
@@ -110,24 +232,54 @@ export async function uploadFile (file: File): Promise<Ref<Blob>> {
   if ('error' in result[0]) {
     throw Error(`Failed to upload file: ${result[0].error}`)
   }
-
-  return id as Ref<Blob>
 }
 
-/**
- * @public
- */
-export async function deleteFile (id: string): Promise<void> {
-  const fileUrl = getFileUrl(id)
-
-  const resp = await fetch(fileUrl, {
-    method: 'DELETE',
+async function uploadFileWithSignedUrl (file: File, uuid: string, uploadUrl: string): Promise<void> {
+  const response = await fetch(uploadUrl, {
+    method: 'POST',
     headers: {
       Authorization: 'Bearer ' + (getMetadata(plugin.metadata.Token) as string)
```

Comment:
Do we need auth when uploading with signed url?

Reply:
Nvm, it's actually a step of obtaining one :)

Reply:
Yep, we need auth to obtain url and do NOT need to upload with signed URL. An attempt to use auth with signed URL will result in 401 or 403 error.